normalizer/normalizer.bal

```diff
@@ -0,0 +1,36 @@
+import ballerina/http;
+
+type ZendeskResponse record {
+    record {|
+        string url;
+        int id;
+        string subject;
+    |} ticket;
+};
+
+http:Client zendeskClient = check new ("http://api.zendesk.com.balmock.io");
+
+service /api/v1 on new http:Listener(8080) {
+    
+    resource function post ticket(@http:Payload json|xml request) returns [int, string, string]|error {
+        json zendeskRequest;
+        if request is json {
+            zendeskRequest = normalize(check request.subject, check request.comment);
+        } else {
+            zendeskRequest = normalize((request/<subject>).data(), (request/<comment>).data());
+        }
+        ZendeskResponse zendeskResponse = check zendeskClient->/api/v2/tickets.post(zendeskRequest);
+        return [zendeskResponse.ticket.id, zendeskResponse.ticket.subject, zendeskResponse.ticket.url];
```

Comment:
Isn't conceptually this same as `return check zendeskClient->/api/v2/tickets.post(zendeskRequest);`? now it returns a json array instead json map. But that is a superficial change. It's still a json.

Reply:
Updated to return the url of the ticket.